core/sail/base/src/main/java/org/eclipse/rdf4j/sail/base/SailSourceConnection.java

```diff
@@ -135,10 +141,17 @@
 	 * @param resolver
 	 */
 	protected SailSourceConnection(AbstractSail sail, SailStore store, FederatedServiceResolver resolver) {
+		this(sail, store, new SimpleEvaluationStrategyFactory(resolver), resolver);
```

Comment:
It is a red flag to me when the same variable is being passed through to two different locations from a constructor like this. 

Is the resolver actually being used here anymore and the fourth argument is just an historical artifact to keep the previous API?